src/main/java/redis/clients/jedis/MultiKeyPipelineBase.java

```diff
@@ -563,4 +563,31 @@
     client.moduleLoad(path);
     return getResponse(BuilderFactory.STRING);
   }
+
+  @Override
+  public Response<List<Long>> bitfield(String key, String... arguments) {
```

Comment:
Please remove this since PipelineBase already implemented it, and single key operations should be there.